### PR TITLE
More redirects from legacy website

### PR DESCRIPTION
**Pull request by ethax-ross**
A few more redirects from the legacy website.

### CODE REVIEW

**Review by leoapost — DISMISSED** [review]

File: config/redirects.yml [hunk 1]
```diff
@@ -266,7 +266,21 @@ redirects:
   "/are-you-ready-to-apply": "/steps-to-become-a-teacher"
   "/whats-your-learning-style": "/ways-to-train"
   "/overseas-applicants": "/international-candidates"
-
+  "/life-as-a-teacher/teaching-vacancies-service": "/my-story-into-teaching"
+  "/five-things-every-prospective-teacher-needs-to-know": "/three-things-to-help-you-get-into-teaching"
+  "/funding-and-salary/overview/scholarships/physics-scholarships": "/salaries-and-benefits"
+  "/explore-my-options/return-to-teaching/return-to-teaching-in-england-from-overseas": "/international-returners"
+  "/regional-make-a-difference-in-bath": ""
```
Comment (peteryates) [inline config/redirects.yml]:
```suggestion
  "/regional-make-a-difference-in-bath": "/"
```

Comment (peteryates) [reply config/redirects.yml]: Not sure if redirecting to "" will just put them at the root of the app, the suggestion might be pointless.

**Review by peteryates — DISMISSED** [review]
Looks good, left one minor suggestion

File: config/redirects.yml (hunk 1 above)
Comment (ethax-ross) [reply config/redirects.yml]: Ah I'm not sure either that was just one I missed 😄

File: config/redirects.yml [hunk 2]
```diff
@@ -266,7 +266,21 @@ redirects:
   "/are-you-ready-to-apply": "/steps-to-become-a-teacher"
   "/whats-your-learning-style": "/ways-to-train"
   "/overseas-applicants": "/international-candidates"
-
+  "/life-as-a-teacher/teaching-vacancies-service": "/my-story-into-teaching"
+  "/five-things-every-prospective-teacher-needs-to-know": "/three-things-to-help-you-get-into-teaching"
+  "/funding-and-salary/overview/scholarships/physics-scholarships": "/salaries-and-benefits"
```
Comment (MylesJarvis) [inline config/redirects.yml]: funding page

File: config/redirects.yml [hunk 3]
```diff
@@ -266,7 +266,21 @@ redirects:
...
+  "/explore-my-options/return-to-teaching/return-to-teaching-in-england-from-overseas": "/international-returners"
+  "/regional-make-a-difference-in-bath": "/"
+  "/getting-school-experience": "/my-story-into-teaching/career-changers/school-experience-helped-me-decide-to-switch"
```
Comment (MylesJarvis) [inline config/redirects.yml]: steps

**Review by MylesJarvis — APPROVED** [review]

**Pull request merged**